qso/handlers.go

```diff
@@ -30,6 +44,16 @@ func (handler *QsoHandler) Get(c *fiber.Ctx) error {
 	return c.JSON(qso)
 }
 
+// Create
+//	@Summary	Create QSO record
+//	@Tags		QSO
+//	@Accept		json
+//	@Produce	json
+//	@Param		request	body		Qso	true	"QSO data"
+//	@Success	200		{object}	Qso
+//	@Failure	400		{object}	object
+//	@Failure	500		{object}	object
+//	@Router		/api/qso [post]
 func (handler *QsoHandler) Create(c *fiber.Ctx) error {
```

Comment:
Is it possible to pre-populate the "Edit Value" model with sample QSO data, so for instance when clicking on the `POST` method, it wouldn't have...

```
{
  "band": 0,
  "callsign": "string",
  "city": "string",
  "country": "string",
  "county": "string",
  "createdAt": "string",
  "dateon": "string",
  "deletedAt": "string",
  "id": 0,
  "mode": "string",
  "name": "string",
  "power": 0,
  "qslr": true,
  "qsls": true,
  "remarks": "string",
  "rstr": 0,
  "rsts": 0,
  "state": "string",
  "timeon": "string",
  "updatedAt": "string"
}```

but rather display something like...
```
{
  "band": 20,
  "callsign": "W1AW",
  "city": "New Haven",
  "country": "USA",
  "county": "New Haven",
  "createdAt": "string",   -- # CAN THIS BE HIDDEN?
  "dateon": "2023-10-28",
  "deletedAt": "string",   -- # CAN THIS BE HIDDEN?
  "id": 0,                           -- # CAN THIS BE HIDDEN?
  "mode": "cw",
  "name": "John Smith",
  "power": 100,
  "qslr": false,
  "qsls": false,
  "remarks": "This is a test contact with W1AW",
  "rstr": 599,
  "rsts": 599,
  "state": "CT",
  "timeon": "16:34:58.947Z",
  "updatedAt": "string" -- # CAN THIS BE HIDDEN?
}
```

Reply:
Apparently you can provide an example values via struct tags, and that's what I did.

As for hiding Gorm fields: that would probably require to have a separate models for Create and Update methods.